packages/sdk/src/fee/utils.ts

```diff
@@ -0,0 +1,37 @@
+import * as ecc from "@bitcoinerlab/secp256k1";
+import { initEccLib, networks, Psbt } from "bitcoinjs-lib";
+
+import { AddressFormat } from "../addresses/types";
+import { P2SH_P2WPKH, P2TR, P2WPKH } from "./__fixtures__/psbt.fixture";
+
+/**
+ * Creates a mock Psbt for testing.
+ *
+ * References:
+ * - https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/test/integration/transactions.spec.ts
+ * - https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/test/integration/taproot.spec.ts
+ *
+ * @returns Psbt
+ */
+function createMockPsbt(format: Exclude<AddressFormat, "legacy">) {
```

Comment:
Mock fns will not be used in the primary flow. Can we move this to test/fixture directory or something?